app/src/main/java/com/keylesspalace/tusky/util/ViewDataUtils.java

```diff
@@ -58,12 +59,24 @@
                 .setApplication(visibleStatus.getApplication())
                 .setStatusEmojis(visibleStatus.getEmojis())
                 .setAccountEmojis(visibleStatus.getAccount().getEmojis())
+                .setCollapsible(collapseLongStatusContent && status.getContent().length() > 500)
```

Comment:
This value is hardcoded in multiple places

Reply:
The issue I ran into is most of those places don't have easy access to preferences (which would be the ideal implementation). I'm looking into what I can access to see if I can easily get the value from the common available items in each method to properly implement this.